docs/UPDATED_CODE_REVIEW_PLAN.md

```
@@ -0,0 +1,1409 @@
+        |> Ash.Query.filter(deleted_at == nil)
+      end
+    end
+  end
+end
+```
+
+---
+
+### 2.5 Domain Consolidation
+
+**Priority: MEDIUM**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)